data/autoscale-formula/TimeBasedAdjustment.txt

```diff
@@ -0,0 +1,5 @@
+$curTime = time();
```

Comment:
Just add a top comment describing briefly what this forumla does(Might be good to have that for all of them)